apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/coverageSearchByPatientId.json

```diff
@@ -908,15 +908,6 @@
         }
       } ],
       "class" : [ {
-        "type" : {
```

Comment:
It's a good idea to look at these files to ensure nothing is being changed accidentally. We shouldn't be seeing these values removed here.

Reply:
yeah, this was part of removing the Group. adding that back for C4DIC should fix this as well